File: app-pacientes-frontend/src/app/data/types/Ubigeo.ts
export class Ubigeo {
  codigo: string;
  descDepartamento: string;
  descProvincia: string;
  descDistrito: string;
  codigoDepartamento: string;
  codigoProvincia: string;
  codigoDistrito: string;

  constructor(
    codigo: string = '',
    descDepartamento: string = '',
    descProvincia: string = '',
    descDistrito: string = '',
    codigoDepartamento: string = '',
    codigoProvincia: string = '',
    codigoDistrito: string = ''
  ) {
    this.codigo = codigo;
    this.descDepartamento = descDepartamento;
    this.descProvincia = descProvincia;
    this.descDistrito = descDistrito;
    this.codigoDepartamento = codigoDepartamento;
    this.codigoProvincia = codigoProvincia;
    this.codigoDistrito = codigoDistrito;
  }
}

File: app-pacientes-backend/src/main/java/pe/tintegro/apppacientesbackend/model/Ubigeo.java
package pe.tintegro.apppacientesbackend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Ubigeo {

    private String codigo;
    private String descDepartamento;
    private String descProvincia;
    private String descDistrito;
    private String codigoDepartamento;
    private String codigoProvincia;
    private String codigoDistrito;
    private Integer estado;

}
